src/routes/index.tsx

```diff
@@ -22,6 +22,7 @@ import {
   LineChartOutlined,
 } from '@ant-design/icons'
 import { MainRoute } from './MainRoute'
+import { DataResearch } from 'src/pages/DataResearch'
```

Comment:
Hmm, I don't know if it is so necessary but the other pages are imported with lazy.